skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldRunConfigurations.kt

```diff
@@ -52,11 +53,26 @@ class SkaffoldDevConfiguration(
     project: Project,
     factory: ConfigurationFactory,
     name: String
-) : RunConfigurationBase(project, factory, name) {
+) : BaseSkaffoldRunConfiguration(project, factory, name) {
 
     override fun getState(executor: Executor, environment: ExecutionEnvironment): RunProfileState? =
         null
 
     override fun getConfigurationEditor(): SettingsEditor<out RunConfiguration> =
         SkaffoldDevSettingsEditor()
 }
+
+/**
+ * Base class for Skaffold run configurations, includes base properties such as Skaffold
+ * configuration file path.
+ */
+abstract class BaseSkaffoldRunConfiguration(
+    project: Project,
+    factory: ConfigurationFactory,
+    name: String
+) : RunConfigurationBase(project, factory, name) {
+
+    /** Persisted Skaffold config file name for Skaffold run configurations. */
```

Comment:
is this a relative path / absolute path? maybe make it clear in the doc

Reply:
absolute, made an update and linked to a source of the path.